.gitignore

```diff
@@ -55,3 +55,6 @@ hw/foundry/
 # Local Tock checkout
 sw/device/tock/tock_local
 sw/device/tock/tock/Cargo.toml
+
+# ROM_EXT signer vendored in dependencies
+sw/host/rom_ext_image_signer/vendored_dependencies
```

Comment:
Is there any way these can be made to live under `third_party/sw/vendored_dependencies`? That would fit with what our best-practices are for project structure.

Reply:
Opened a follow-up issue: https://github.com/lowRISC/opentitan/issues/4351